test/NuGet.Core.FuncTests/Msbuild.Integration.Test/MsbuildRestoreTaskTests.cs

```diff
@@ -1629,17 +1592,10 @@ public async Task MsbuildRestore_PackageNamespaceLongerMatches_NoNamespaceMatche
                 var result = _msbuildFixture.RunMsBuild(pathContext.WorkingDirectory, $"/t:restore -v:d {pathContext.SolutionRoot} /p:RestorePackagesConfig=true", ignoreExitCode: true);
 
                 // Assert
-                Assert.True(result.ExitCode == 0);
-                var contosoRestorePath = Path.Combine(projectAPackages, packageOpenSourceContosoMvc.ToString(), packageOpenSourceContosoMvc.ToString() + ".nupkg");
-                using (var nupkgReader = new PackageArchiveReader(contosoRestorePath))
-                {
-                    var allFiles = nupkgReader.GetFiles().ToList();
-                    // Assert correct package was restored.
-                    Assert.Contains("lib/net461/openA.dll", allFiles);
-                }
-
-                Assert.True(result.ExitCode == 0);
+                Assert.Equal(1, result.ExitCode);
                 Assert.Contains("Package namespace match not found for package ID 'My.MVC.ASP'", result.Output);
+                Assert.Contains("warning : Unable to find version '1.0.0' of package 'My.MVC.ASP'.", result.Output);
+                Assert.Contains("error MSB4181: The \"RestoreTask\" task returned false but did not log an error.", result.Output);
```

Comment:
This seems wrong doesn't it? 
It should be logging an error.

Reply:
@nkolev92 
It looks we have another bug not related to package namespace in our code. I debugged the it, somehow we're logging exception as warning, it should have logged as error.
![image](https://user-images.githubusercontent.com/8766776/130543712-ce6e7189-b741-4373-9932-3a79e004f9e2.png)

Would you like me create bug for it?

Reply:
@nkolev92 
I created follow up issue https://github.com/NuGet/Home/issues/11179 for above.